### ISSUE TITLE
Custom COMMIT_MESSAGE_SEPARATOR

### ISSUE THREAD

**Issue by rickythefox** [Gherciu/commitlint-jira#11]
Is there a way to configure the commit separator in commitlint.config.js? For example I'd like mine to be ': ' (with a space after).

**Comment by Gherciu** [comment]
@rickythefox  At the moment this package doesn't have a way to configure commit separator, but in next release, this possibility will be added, I have already raised an "enhancement" for that https://github.com/Gherciu/commitlint-jira/issues/4 . Next release has planned for the end of January. I will keep open this issue until the next release and after publishing the next version I will notify you. Thanks.

**Comment by Gherciu** [comment]
Hi, @rickythefox  this issue is fixed as part of `1.3.0`, to install it just run
```bash
npm i -D commitlint-plugin-jira-rules@1.3.0 commitlint-config-jira@1.3.0
```
And now your config file should look like this if you want to customise/override `jira-commit-message-separator` to be ": "
```js
// commitlint.config.js
module.exports = {
  plugins: ['commitlint-plugin-jira-rules'],
  extends: ['jira'],
  rules: {
    'jira-commit-message-separator': [2, 'always', ': '],
  },
}
```
